mediaflow_proxy/routes/speedtest.py

```
@@ -1,43 +1,41 @@
+        # Get test URLs and user info
+        test_urls, user_info = await provider_impl.get_test_urls()
+        config = await provider_impl.get_config()
+
+        return BrowserSpeedTestConfig(
+            provider=test_request.provider,
+            test_urls=test_urls,
+            test_duration=config.test_duration,
+            user_info=user_info.model_dump() if user_info else None,
+        )
+    except Exception as e:
+        raise HTTPException(status_code=400, detail=str(e))
```

Ruff (0.11.9)

Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)